EndlessClient/Dialogs/TextMultiInputDialog.cs

```diff
@@ -34,6 +34,7 @@ public struct InputInfo
         private readonly Rectangle _backgroundSourceRectangle;
 
         private readonly Vector2 _bottomOverlayDrawPosition;
+        private readonly float _buttonYOffset;
```

Comment:
This can be a local variable in the constructor

Reply:
I'd also maybe lean towards determining the position of the buttons in the `switch` block and then using those `Vector2`s directly in the constructors for the ok/cancel buttons